apps/studio/src/components/export/ExportModal.vue

```diff
@@ -200,9 +212,22 @@ export default {
     defaultFileName () {
       const now = new Date();
       const formatted = dateFormat(now, 'yyyy-mm-dd_HHMMss')
-      const schema = this.table.schema ? `${this.table.schema}_` : ''
       const extension = this.selectedExportFormat.key
-      return `${schema}${this.table.name}_export_${formatted}.${extension}`
+      let fileName;
+      if (this.table) {
+        const schema = this.table.schema ? `${this.table.schema}_` : ''
+        const extension = this.selectedExportFormat.key
+        fileName = `${schema}${this.table.name}_export_${formatted}.${extension}`
+      } else {
+        // sanitize query name for use as filename
+        console.log('this.query_name: ', this.query_name)
```

Comment:
nitpick - `query_name` should be `queryName`.

Also -- can you remove these console.logs?

Reply:
these nits picked in 13a301a